cdap-sentry/cdap-sentry-extension/README.rst

```diff
@@ -105,6 +105,9 @@ extension:
      * - Parameter
        - Description
        - Default Value
+     * - ``security.authorization.enabled``
```

Comment:
can we describe this in a separate section outside this table? This table was supposed to contain only the properties that the security extension uses.